### PR TITLE
[RU-002] new Title and Media links

### PR DESCRIPTION
**Pull request by TimoPeraza**
Addding a new Title with the logo and new links to social medias following the design:
https://www.figma.com/file/TSWNtfn3NFEZhBmSzPG116/%5BRuby-UI%5D-Lading-Design?node-id=73%3A243&mode=dev
Screenshot 2024-01-30 at 16 52 50

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ _site
 .jekyll-cache
 .jekyll-metadata
 vendor
+assets/.DS_Store
+assets/images/.DS_Store
```
Comment (elcuervo) [inline .gitignore:8]: You can just:
```suggestion
.DS_Store
```

**Comment by santib** [comment]
There are some responsiveness issues. Maybe it'd be good to build it from the start, otherwise it will need a re-work. What do you think?

**Review by elcuervo — COMMENTED** [review]
Love it. We might want to chain some PRs to this one before merging so it does not look so "broken": https://f59727f3.ruby-uy.pages.dev/
Some markdown was not processed correctly I think

File: .gitignore (hunk 1 above)
Comment (santib) [reply .gitignore:8]: yeah or just add it to your global `~/.gitignore` not to the project one.

File: index.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<div class='oldSection'>
```
Comment (delbetu) [inline index.md:1]: everything inside this div should be indented one level deeper, right?

Comment (TimoPeraza) [reply index.md:1]: mm one level deeper from where? Anyway, I think this oldSection will disappear, I added it just to show while we are doing the rest of the page.

**Pull request closed without merging**